### PR TITLE
Fix for on(:long_press) firing twice.

### PR DESCRIPTION
**Pull request by markrickert**
Whoopsie.

dididothat

### CODE REVIEW

File: motion/ruby_motion_query/event.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def initialize(sender, event, block)
 
     def handle_gesture_or_event(gesture)
       # Don't fire :long_press events twice.
-      return if @sdk_event_or_recognizer == VIEW_GESTURES[:long_press] && gesture.state == UIGestureRecognizerStateEnded
+      return if gesture.class == VIEW_GESTURES[:long_press] && gesture.state != UIGestureRecognizerStateBegan
```
Comment (squidpunch) [inline motion/ruby_motion_query/event.rb]: just curious, why the change from `@sdk_event_or_recognizer` to `gesture.class` ?  I think I am following whats going on here, but want to make sure :)

Comment (twerth) [inline motion/ruby_motion_query/event.rb]: Good question

Comment (markrickert) [inline motion/ruby_motion_query/event.rb]: It was a good question and I have no good answer. They do the same thing here. We can change it back to `@sdk_event_or_recognizer` if you'd like, however I think that checking the class of the var passed to the method is probably safer than checking an instance variable.

Comment (squidpunch) [inline motion/ruby_motion_query/event.rb]: Thanks @markrickert.  It looked fine to me I was just verifying it was as you said the same thing and not what caused the bug, to be aware of if we did the same in other areas, etc :)
